site/_posts/2018-02-14-the-pathname-to-discovering-a-memory-leak-in-ruby.html.md

```diff
@@ -0,0 +1,429 @@
+---
+title: The Path[name] to discovering a memory leak in Ruby
+author: NickLaMuro
+date: 2018-02-14
+tags: ruby performance
+comments: true
+published: true
+---
+
+
+_**tl;dr;** we found a [memory leak in
```

Comment:
nitpicking, but it's usually abbreviated **tl;dr** or **tldr;** but not **tl;dr;** :sweat_smile:

Reply:
TIL, thanks!

Though, if the size of this blog post wasn't any indication, I typically am not one that subscribes to brevity (@chrisarcand would know what I am talking about 😏 ).

Reply:
I love reading epic posts like yours, I just don't have the talent to write them. However I'm well-versed in internet slang/acronyms :rofl: